### ISSUE TITLE
[Accessibility] [HC] The little triangle of "toolbar options" button in ToolStrip is hard to visible when focusing on it

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#7724]
### .NET version

8.0.100-alpha.1.22451.15

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, not a regression issue

### Issue description

The little triangle of "Toolbar options" button in ToolStrip is hard to visible on Windows 11 high contrast themes and Windows 10 high contrast black/white themes

**Windows 11 HC Aquatic/Desert/Dusk/Night Sky testing results:**
![image](https://user-images.githubusercontent.com/26474449/188071242-fa1ae9d3-b934-4c5e-b1fb-da2e3749d16e.png)

**Windows 11 default theme:**
![image](https://user-images.githubusercontent.com/26474449/188071847-19f345ea-ae4a-4f1c-9cac-4509a242a28f.png)

**Windows 10 HC black/white/#1/#2 testing result:**
![image](https://user-images.githubusercontent.com/26474449/188071523-2223bd50-6189-454f-9736-ec7e681017af.png)

**Windows 10 default theme:**
![image](https://user-images.githubusercontent.com/26474449/188071884-9d3db6bf-becd-4fd9-aa16-f09b00aebdf2.png)

### Steps to reproduce

Sample application: 
[WinFormsApp9.zip](https://github.com/dotnet/winforms/files/9475449/WinFormsApp9.zip)

**Comment by merriemcgaw** [comment]
That glyph should come from a standard System color. It's also pretty clear we don't meet contrast minimums here either.

**Comment by Amy-Li03** [comment]
Verified this on .NET 8.0 latest build: 8.0.100-alpha.1.22552.11, issue was fixed on Win10 and Win11 OSes.
![image](https://user-images.githubusercontent.com/62929087/199674394-152e7dfb-47b8-4106-af4c-089ba125f5f9.png)
![image](https://user-images.githubusercontent.com/62929087/199674792-f603475a-1544-44d7-a5cb-8343a00f9eba.png)
